packages/react/src/hooks/actions/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const HUB_AUTH_SIGNOUT_NAMESPACE = 'AuthSignOut';
+export const HUB_NAVIGATE_NAMESPACE = 'navigate';
```

Comment:
I was wondering which case we should use for the channel name. Would like to be consistent with the existing ones we have, like:
```
core
auth
api
analytics
interactions
pubsub
storage
xr
datastore
```

Reply:
Yeah, I'm thinking we should add a new channel name `actions`

Reply:
Having `actions` channel and then differentiate actions through `event`? Like `XXX_Started` and `XXX_Finished`